pkgs/applications/science/logic/klee/default.nix

```diff
@@ -14,6 +12,27 @@
 , gtest
 , lit
 
+# The chosen version of LLVM.
+, chosenLLVMPackages ? let
```

Comment:
Probably should put this in a let block.

Reply:
Just defaulted it to null so they can still override it if they really want to, and moved this and the below to the let block.